tests/stubtest_allowlists/py3_common.txt

```diff
@@ -26,11 +26,11 @@ _collections_abc.Set.__rxor__
 _csv.Dialect.__init__  # C __init__ signature is inaccurate
 _socket.*
 _threading_local.local.__new__
-_weakref.ref.__call__
-abc.ABCMeta.__new__  # pytype wants the parameter named cls and not mcls
+_weakref.ref.*  # Alias for _weakref.ReferenceType, problems should be fixed there
 _weakref.CallableProxyType.__getattr__  # Should have all attributes of proxy
 _weakref.ProxyType.__getattr__  # Should have all attributes of proxy
 _weakref.ReferenceType.__call__  # C function default annotation is wrong
+_weakref.ReferenceType.__init__  # Runtime defines __new__ but stubtest thinks __init__ is also defined.
```

Comment:
This is a guess. But if I'm not correct here, I think the best thing to do here is to have a white lie in the stub anyway.